wp-content/plugins/core/src/Panels/Types/Hero.php

```diff
@@ -17,8 +17,8 @@ class Hero extends Panel_Type_Config {
 	const FIELD_IMAGE                        = 'image';
 	const FIELD_CTA                          = 'cta';
 	const FIELD_TEXT_COLOR                   = 'text_color';
-	const FIELD_TEXT_LIGHT                   = 't-content--light';
-	const FIELD_TEXT_DARK                    = 't-content--dark';
+	const FIELD_TEXT_LIGHT                   = 't-theme--light';
+	const FIELD_TEXT_DARK                    = 't-theme--dark';
```

Comment:
Nothing to update right now, just noting I'd like to probably migrate skins/themes to have their own prefixed namespace once we land on final approach.

Reply:
:+1: Agreed.